traffic-dashboard/src/components/sidebar/sidebar.tsx

```diff
@@ -62,6 +62,9 @@ export const SideBar: React.FC = () => {
             onClick={() => {
               navigate("/home");
             }}
+            selected={
+              "/home" === location.pathname || "/" === location.pathname
```

Comment:
Please use the routes const for the path names.
Consider moving the logic to a function to avoid repeated code. And that we would be able to change the logic in a single place.

 https://github.com/sharkio-dev/sharkio/blob/8d6f86f4073d9e61a0a6a2c6d70619cf3737a854/traffic-dashboard/src/constants/routes.ts#L1-L14